app/src/integrationTest/kotlin/maru/app/MaruFollowerTest.kt

```diff
@@ -159,6 +162,7 @@ class MaruFollowerTest {
         engineApiRpc = followerStack.besuNode.engineRpcUrl().get(),
         dataDir = followerStack.tmpDir,
         validatorPortForStaticPeering = validatorStack.p2pPort,
+        syncingConfig = MaruFactory.defaultSyncingConfig.copy(desyncTolerance = desyncTolerance),
```

Comment:
iiuc, the `desyncTolerance` being passed above is 0UL, we should be able to skip the above line as the default `syncingConfig` is already have `desyncTolerance` set as 0UL